examples/juniper/cptx-ixia/cptx-ixia.pb.txt

```diff
@@ -0,0 +1,110 @@
+name: "ixia-c"
+nodes: {
+    name: "otg"
+    vendor: KEYSIGHT
+    version: "0.0.1-3423-local"
+    services: {
+        key: 443
+        value: {
+            name: "ixia-c"
+            inside: 443
+            outside: 0
```

Comment:
please omit the outside ports for the otg node

Reply:
Done